packages/tupaia-web/src/features/DashboardItem/ExpandItemButton.tsx

```diff
@@ -54,8 +53,8 @@ const ExpandButtonText = styled.span`
 const ZoomInIcon = styled(MuiZoomIcon)`
   margin-right: 1rem;
   @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
-    width: 3rem;
-    height: 3rem;
+    width: 1.5rem;
+    height: 1.5rem;
```

Comment:
These look quite small to me. If you haven't already, can you send Flic a screenshot of before and after this one change, to make sure it's correct?